src/App.tsx

```diff
@@ -27,13 +27,14 @@ import { SourceTargetContextProvider } from './contexts/SourceTargetContextProvi
 import { TransactionContextProvider } from './contexts/TransactionContext';
 import { useConnections } from './hooks/useConnections';
 import Main from './screens/Main';
+import { isEmpty } from 'lodash';
 
 const [connectionDetails1, connectionDetails2] = substrateProviders();
 
 function App() {
-  const { connections, apiReady } = useConnections([connectionDetails1, connectionDetails2]);
+  const { connections } = useConnections([connectionDetails1, connectionDetails2]);
 
-  if (!apiReady) {
+  if (isEmpty(connections)) {
```

Comment:
Under what circumstances can this happen?

Reply:
When chain1Configs or chain2Configs is empty -> meaning for each chain - when the API is not connected:
https://github.com/paritytech/parity-bridges-ui/blob/5e8851725cb774981b88ef3c4fea8732978ffede/src/hooks/useApiConnection.ts#L81
If this does not happen, then getChainConfigs will not run, and thus configs will be empty...